App/Navigation/AppNavigation/Primary/TabNavigator/index.tsx

```diff
@@ -37,7 +37,7 @@ const nav = createBottomTabNavigator(
     },
     animationEnabled: false,
     swipeEnabled: false,
-    initialRouteName: 'Wallet'
+    initialRouteName: 'Threads'
```

Comment:
Just making sure you want this in there? Thought you wanted to highlight Wallet.